conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Plain.java

```diff
@@ -33,22 +29,14 @@
 import java.util.function.Function;
 import javax.annotation.Nullable;
 
-/** Serialization functionality consumed by generated code. */
-public interface SerDe {
-
-    /** Create a {@link Serializer} for the requested type. Serializer instances should be reused. */
-    <T> Serializer<T> serializer(TypeToken<T> type);
-
-    /** Create a {@link Deserializer} for the requested type. Deserializer instances should be reused. */
-    <T> Deserializer<T> deserializer(TypeToken<T> type);
-
-    /** Serialize a {@link BinaryResponseBody} to <pre>application/octet-stream</pre>. */
-    void serialize(BinaryResponseBody value, HttpServerExchange exchange) throws IOException;
-
-    /** Reads an {@link InputStream} from the {@link HttpServerExchange} request body. */
-    InputStream deserializeInputStream(HttpServerExchange exchange);
+/**
+ * Provides functionality to parse supported types using the
+ * <a href="https://github.com/palantir/conjure/blob/master/docs/spec/wire.md#6-plain-format">Conjure PLAIN format</a>.
+ *
+ * These utilities are used to parse HTTP path, query, and header parameter values.
+ */
+public interface Plain {
```

Comment:
PlainSerDe?

Reply:
There's no `Ser`, only `De` for plain fields on servers, but if we did add serialization we would put it here. 👍